set.mm

```diff
@@ -650059,7 +650059,7 @@ not even needed (it can be any class).  (Contributed by Glauco
       $.
   $}
 
-  $( ` RR^ 2 ` is a subset of ` CC^ 2 ` .  (Contributed by Glauco Siliprandi,
+  $( ` RR^ 2 ` is a subset of CC^ 2.  (Contributed by Glauco Siliprandi,
```

Comment:
I think the symbols are misused here: does ` RR^ 2 ` is also not defined. Therefore, I propose to change the comment to "The cartesian product of ` RR ` with itself is a subset of the cartesian product of ` CC ` with itself. " (BTW: sometimes Cartesian is written with an upper case C, sometimes with a lower case c...).

Reply:
The file is encoded in ASCII, but if really a non-ASCII character is needed, then sometimes the HTML encoding of this character is used.  Like in "B&eacute;zout's identity", standing for "Bézout's identity".  A superscript 2 is likewise displayed by &#178; (see https://unicode-table.com/de/00B2/), and the set of real numbers ℝ by &#8477;, so we end up with &#8477;&#178; for ℝ².  Perfect display on the HTML page, barely understandable when viewed in an editor.

There was a reason why Norm allowed quoting math expressions in comments.  If at all possible I suggest keeping the quoted style.  In the long run I suggest to fully support Unicode (UTF-8 is a good candidate).

Reply:
I removed these backticks since `CC^` is not (yet) a defined symbol. But as @avekens wrote, the theorem is not about `RR^` anyway. So I will comment "The cartesian square of RR is included in the cartesian square of CC."

Reply:
As for @wlammen's second comment: thanks for the info.  I think that indeed quoting math symbols is comments is good because it keeps the file set.mm plain ASCII, with the unicode definitions of symbols gathered in a common place (the $t comment).